### PR TITLE
Update URLs and some cleanups

### PR DESCRIPTION
**Pull request by atouchet**
Update repo URL, remove Travis reference, and some minor Readme cleanup.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ library.
 
 ## [API Documentation](https://docs.rs/bincode/)
 
-## Bincode in the wild
+## Bincode in the Wild
```
Comment (VictorKoenders) [inline readme.md:23]: I don't think the word `wild` needs to be capitalized

Comment (atouchet) [reply readme.md:23]: That's just to be consistent with the other first level headers in the Readme.

**Review by VictorKoenders — APPROVED** [review]

**Comment by VictorKoenders** [comment]
Looks good to me, thanks!

**Comment by VictorKoenders** [comment]
(Waiting for #379 to merge to fix the CI pipeline)

**Comment by VictorKoenders** [comment]
#379 has landed, could you fix the merge issues? I'll merge this afterwards

**Comment by VictorKoenders** [comment]
Thanks!

**Pull request merged**